policy/modules/contrib/thumb.te

```diff
@@ -46,6 +46,7 @@ allow thumb_t self:sem create_sem_perms;
 allow thumb_t self:cap_userns { net_admin setpcap sys_admin sys_ptrace };
 allow thumb_t self:process setcap;
 allow thumb_t self:user_namespace create;
+allow thumb_t tmp_t:dir mounton;
```

Comment:
It is no allowed to refer to types from other modules. If an appropriate interface does not exist, you need to create one - see https://github.com/fedora-selinux/selinux-policy/pull/2982